src/components/InternalDropdown/pageObject/index.js

```diff
@@ -75,6 +84,73 @@ class PageInternalDropdown {
             bottom: y + height,
         };
     }
+
+    /**
+     * Returns true when the arrow to scroll down exits, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    arrowDownExists() {
+        return $(this.rootElement)
+            .$('[data-id="internal-dropdown-arrow-down"]')
+            .isExisting();
+    }
+
+    /**
+     * Returns true when the arrow to scroll down exits, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    arrowUpExists() {
+        return $(this.rootElement)
+            .$('[data-id="internal-dropdown-arrow-up"]')
+            .isExisting();
+    }
+    /**
+     * Returns true when the search no results found, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    emptyMessageExist() {
+        return $(this.rootElement)
+            .$('p*=Our robots')
+            .isExisting();
+    }
+
+    /**
+     * Clicks the input element.
+     * @method
+     */
+    clickInputSearch() {
+        $(this.rootElement)
+            .$('input[type="search"]')
+            .click();
+    }
+    /**
+     * Type in the input element.
+     * @method
+     * @param {string} value - The value to type in the input element.
+     */
+    setSearchCriteria(value) {
+        $(this.rootElement)
+            .$('input[type="search"]')
+            .setValue(value);
+    }
+    /**
+     * Returns true when the menu item is visible inside the menu container.
+     * @method
+     * @returns {bool}
+     */
+    isVisible() {
+        return $(this.rootElement).isDisplayedInViewport();
+    }
+    /**
+     *  Wait until the option is visible.
+     * @method
+     */
+    waitUntilIsVisible() {
+        browser.waitUntil(() => this.isVisible());
+    }
```

Comment:
in this case we don't need this both last methods, for example when use it in picklist is where there is a functionality where the menu isVisible and hidden, but here there is not such functionality